app/main/helpers/shared_helpers.py

```diff
@@ -13,3 +10,20 @@ def get_one_framework_by_status_in_order_of_preference(frameworks, statuses_in_o
         for framework in frameworks:
             if framework.get('status') == status:
                 return framework
+
+
+def construct_url_from_base_and_params(base_url, query_params):
```

Comment:
Nice - this makes much more sense like this

Reply:
👍